ballerina/pipe.bal

```diff
@@ -35,8 +35,9 @@ public isolated class Pipe {
         'class: "io.xlibb.pipe.Pipe",
         paramTypes: ["java.lang.Long"]
     } external;
-    
+
     public isolated function generatePipeWithTimer(int 'limit, Timer timer) = @java:Method {
+        name: "generatePipe",
```

Comment:
Can't we rename the method so that we can omit this field?

Reply:
Same goes here as in https://github.com/xlibb/module-pipe/pull/36#discussion_r1754319979